tests/conftest.py

```diff
@@ -17,7 +17,7 @@ def pyramid_settings():
         "checkmate_ignore_reasons": None,
         "checkmate_allow_all": False,
         "enable_front_page": True,
-        "data_directory": ".devdata",
+        "data_directory": "tests/data_directory",
```

Comment:
This change is needed because this directory doesn't exist in CI, and so the functests fail to start. We don't actually need much to be in those files on startup, but we do need them to exist.